web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v304/migrate-default.sql

```diff
@@ -1,2 +1,4 @@
 UPDATE Settings SET value='3.0.4' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
+
+ALTER TABLE HarvesterData RENAME COLUMN "key" TO keyvalue;
```

Comment:
The merged was done after 3.0.4 was released. So those statements should move to 3.0.5 I guess.

Would be good to also merge 3.0.x to develop after those changes.